### DISCUSSION TITLE
posts/trying-to-build-a-micro-journalling-app/

### DISCUSSION THREAD

**Discussion by giscus** [nerdymomocat/nerdymomocat.github.io#7] [Announcements]
# posts/trying-to-build-a-micro-journalling-app/

I'm trying to create a highly personal and experimental micro journal app tailored to my unique preferences, circumventing common app limitations and Notion's page layout constraints, with a focus on free-form input and LLM-powered structured data extraction in a timeline format with blocks as the main input format.

https://nerdymomocat.github.io/posts/trying-to-build-a-micro-journalling-app/

**Comment by Youhan** [comment]
Based on your research on existing note taking apps what do you think about this one? https://lazy.so/

**Reply by nerdymomocat** [reply]
I have seen this before. I don't have access to it; and I think while it does do some parts of metadata extraction that I would want -- I am not looking for another platform locked app.

**Reply by Youhan** [reply]
Makes sense.
